scripts/shared.js

```diff
@@ -1203,3 +1236,47 @@ export function buildAdBlock(unitId, type, fixedHeight = false) {
   const range = document.createRange();
   return range.createContextualFragment(rightAdHTML);
 }
+
+/**
+ * Build previous next buttons
+ */
+
+export function prevNextBtn(articleCount) {
```

Comment:
Since the styling for these components are in the category template, would it make more sense for this code to be in the category template, too? Or are there other places where we'd use this?

Reply:
@mfrisbey, that's the thought. Having it in shared.js makes this method available for future re-use. 
In case, CRN/ensemble want to implement prev/next button, they can re-use this code, or copy-paste it into a separate block.